crates/autopilot/src/infra/solvers/dto/mod.rs

```diff
@@ -0,0 +1,4 @@
+pub mod quote;
```

Comment:
Why are all these new files? Can't they be moved from existing dtos? Can we maybe leave that to a different PR (to keep the size of this one manageable)?